Gems/EMotionFX/Code/EMotionFX/Source/BlendTreeBlendNNode.cpp

```diff
@@ -645,7 +645,7 @@ namespace EMotionFX
         data->SetTrajectoryDeltaMirrored(delta);
     }
 
-    bool BlendTreeBlendNNode::VersionConverter(AZ::SerializeContext& context, AZ::SerializeContext::DataElementNode& classElement)
+    static bool BlendTreeBlendNNodeVersionConverter(AZ::SerializeContext& context, AZ::SerializeContext::DataElementNode& classElement)
```

Comment:
I wonder if we should just be removing these version converters entirely, since they aren't compatible with prefabs anyways...

Reply:
Yes, as long as the objectstream format is not being used anymore (ie, its json now) or its a product (which gets remade whenever a version is bumped).  The objectsream VersionConverters will not acutally port to json format, even if someone made the effort, the actual structure is different, so they are not "save-able".

However, some files still use objectstream, as the source (right?) and if these objects appear in there, the version converter must be maintained.